packages/paste-core/components/menu/src/index.tsx

```diff
@@ -49,9 +51,12 @@ const StyledMenuItem = React.forwardRef<HTMLDivElement, BoxProps>(({children, ..
   return (
     <Box
       {...safelySpreadBoxProps(props)}
+      as="a"
+      display="block"
       padding="space30"
       paddingLeft="space50"
       paddingRight="space50"
+      textDecoration="none"
```

Comment:
Why do we need this both on here and on the inner Text component?

Reply:
Being set here and on child is potentially an oversight, but I was trying to keep in line with the existing design constraint, but thinking about it more I'm convinced we have wriggle room in the constraint.

The problem we face in this particular instance is that to get the browser link behaviours, cmd + click open in new tab for example, you need to use a _traditionally_ inline text element as a block element. 

Browser defaults are annoying in that you can't undo text decoration from a parent node, and links get underlined by the browser. https://codepen.io/SiTaggart/pen/ZEbNbQV

So that leads you in this conflict between box and text style separation. I need to set what is _traditionally_ a text based style on the box level element. The wriggle room we have here though is the reclassification of anchor in HTML5 to have a ["transparent" content model](https://www.w3.org/TR/html52/textlevel-semantics.html#the-a-element). Meaning it is now both inline and block at the same time. For me, that means styles specific to anchors do need to cater for Box and Text, it is both, and therefore I'm fine with anchor specific styling to be present on Box and Text.

I'll just remove the `textDecoration` on the child.

Reply:
Further to this: If you _don't_ set the textDecoration on the node that sets the text color, the underline color doesn't match. So for now it still needs to appear at both levels.

I don't necessarily feel comfortable having to add color to box just to accommodate that, given the limited impact of having an "unset styling property" on the parent, and a set styling prop on the text node that takes color. The change to box is then at least scoped to the purpose, accommodating the transparent content modal of the anchor.

Reply:
Cool, I'm fine with this.